server/src/main/java/de/uftos/config/CustomPostgreSQLDialect.java

```diff
@@ -0,0 +1,20 @@
+package de.uftos.config;
+
+import org.hibernate.boot.model.FunctionContributions;
+import org.hibernate.dialect.PostgreSQLDialect;
+
+public class CustomPostgreSQLDialect extends PostgreSQLDialect {
+  public CustomPostgreSQLDialect() {
+    super();
+  }
+
+  @Override
+  public void initializeFunctionRegistry(FunctionContributions functionContributions) {
+    super.initializeFunctionRegistry(functionContributions);
+    var functionRegistry = functionContributions.getFunctionRegistry();
+    functionRegistry.registerPattern(
+        "tsvector_match",
+        "(?1 @@ ?2)"
```

Comment:
What does this line mean?

Reply:
Again, as can be seen in the description, the actual search looks like `WHERE something @@ to_tsquery` and since you cannot write this in specifications ootb, I added a function that takes to arguments (1 and 2) and builds `?1 @@ ?2`